ui/admin/app/components/form/user/add-accounts/index.hbs

```diff
@@ -30,11 +30,15 @@
         {{#each this.filteredAccounts as |account|}}
           <body.row as |row|>
             <row.cell>
-              <form.checkbox
-                @label={{account.id}}
-                @description={{account.description}}
-                @onChange={{fn this.toggleAccount account}}
-              />
+              {{!TODO: Remove the addAccount ability check during Phase 2}}
+              <Hds::Form::Checkbox::Field
+                disabled={{if (can 'addAccount account' account) false true}}
```

Comment:
Not sure which one reads better or if we have a convention but we also could have used `not` instead of a ternary